build.gradle

```diff
@@ -69,7 +69,7 @@ dependencies {
 }
 
 shadowJar {
-    archiveFileName = 'addressbook.jar'
+    archiveFileName = 'volunsync.jar'
```

Comment:
I believe think `volunSync-v1.3.jar` will make it clearer, as it follows the standard `{archiveBaseName}-{archiveVersion}.jar` format. This will also help avoid confusion in the future when we have more JAR files. Other than that, all lgtm!